btr-api/src/btr_api/schemas/SignificantIndividualsFiling.json

```diff
@@ -0,0 +1,176 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "FilingActionE": {
+      "enum": ["add", "edit", "remove"],
+      "type": "string"
+    },
+    "ControlOfDirectorsI": {
+      "properties": {
+        "directControl": {
+          "type": "boolean"
+        },
+        "indirectControl": {
+          "type": "boolean"
+        },
+        "significantInfluence": {
+          "type": "boolean"
+        },
+        "inConcertControl": {
+          "type": "boolean"
+        }
+      },
+      "type": "object",
+      "required": ["directControl", "indirectControl", "significantInfluence", "inConcertControl"]
+    },
+    "ControlOfSharesI": {
+      "properties": {
+        "registeredOwner": {
+          "type": "boolean"
+        },
+        "beneficialOwner": {
+          "type": "boolean"
+        },
+        "indirectControl": {
+          "type": "boolean"
+        },
+        "inConcertControl": {
+          "type": "boolean"
+        }
+      },
+      "type": "object",
+      "required": ["registeredOwner", "beneficialOwner", "indirectControl", "inConcertControl"]
+    },
+    "CitizenshipTypeE": {
```

Comment:
In all our schemas, and in openownership the elements are all camelCase.
Please follow that standard